### ISSUE TITLE
botocore VersionConflict issue

### ISSUE THREAD

**Issue by anthoneous** [venth/aws-adfs#68]
It appears that after upgrading from 0.4.4 to 0.4.7 makes aws-adfs no longer work. 

Downgrading to 0.4.4 (last working version) doesn't resolve the issue. 

Setting up a new python environment doesn't resolve the issue either.

```
: > aws-adfs login
Traceback (most recent call last):
  File "/Users/.pyenv/versions/2.7.14/bin/aws-adfs", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/Users/.pyenv/versions/2.7.14/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3142, in <module>
    @_call_aside
  File "/Users/.pyenv/versions/2.7.14/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3126, in _call_aside
    f(*args, **kwargs)
  File "/Users/.pyenv/versions/2.7.14/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3155, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/Users/.pyenv/versions/2.7.14/lib/python2.7/site-packages/pkg_resources/__init__.py", line 666, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/Users/.pyenv/versions/2.7.14/lib/python2.7/site-packages/pkg_resources/__init__.py", line 679, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/Users/.pyenv/versions/2.7.14/lib/python2.7/site-packages/pkg_resources/__init__.py", line 872, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (botocore 1.8.20 (/Users/.pyenv/versions/2.7.14/lib/python2.7/site-packages), Requirement.parse('botocore<1.9.0,>=1.8.22'), set(['boto3']))

> pip freeze
asn1crypto==0.24.0
astroid==1.5.3
aws-adfs==0.4.7
awscli==1.14.16
backports.functools-lru-cache==1.4
blessings==1.6
boto==2.48.0
boto3==1.5.8
botocore==1.8.20
cachetools==2.0.1
certifi==2017.11.5
cffi==1.11.2
chardet==3.0.4
click==6.7
colorama==0.3.7
configparser==3.5.0
cryptography==2.1.4
docutils==0.14
enum34==1.1.6
futures==3.2.0
google-api-python-client==1.6.4
google-auth==1.1.1
google-cloud-bigquery==0.27.0
google-cloud-core==0.27.1
google-resumable-media==0.3.1
googleapis-common-protos==1.5.3
httplib2==0.10.3
idna==2.6
ipaddress==1.0.19
isort==4.2.15
jmespath==0.9.3
lazy-object-proxy==1.3.1
lxml==4.1.1
mccabe==0.6.1
oauth2client==4.1.2
prettytable==0.7.2
protobuf==3.4.0
py==1.4.34
pyasn1==0.4.2
pyasn1-modules==0.1.5
pycparser==2.18
pylint==1.7.4
pyOpenSSL==17.5.0
pytest==3.2.3
python-dateutil==2.6.1
PyYAML==3.12
requests==2.18.4
rsa==3.4.2
s3transfer==0.1.12
singledispatch==3.4.0.3
six==1.11.0
uritemplate==3.0.0
urllib3==1.22
wrapt==1.10.11
```

**Comment by mark-kawakami-zefr** [comment]
Note that this isn't the same version conflict (though it is for the same dependency) as was resolved with #65.

Also, in this case, I'm seeing this running on a Mac

**Comment by anthoneous** [comment]
It appears that the following versions are necessary to get a working version of aws-adfs working again.

```
aws-adfs==0.4.3
awscli==1.11.154
boto3==1.4.7
botocore==1.7.12
```

**Comment by venth** [comment]
Unfortunately I set version range for boto3 too wide. I've released version: 0.4.8 with:
* aws-cli version: 1.14.18
* boto3 version: 1.5.8 and
* botocore version: 1.8.22

**Comment by anthoneous** [comment]
Issue is resolved in the latest version.
